lib/enkf/ert_workflow_list.cpp

```diff
@@ -196,52 +196,9 @@ const workflow_job_type * ert_workflow_list_get_job( const ert_workflow_list_typ
     return workflow_joblist_get_job(workflow_list->joblist, job_name);
 }
 
-/**
-   This function will create the most specific matching filename
-   corresponding to the @root_name input. I.e. it will look for
-   filename in this order:
 
-     ROOT@X.Y.Z, ROOT@X.Y, ROOT@X, ROOT
-
-   And return the first name corresponding to an existing file. If no
```

Comment:
Gee - this was much work at one stage; I would not have heart to kill it myself - but it sure is good to see it go :+1: